themebuilder/package.json

```diff
@@ -24,5 +25,6 @@
     "mock-require": "^3.0.2",
     "less": "^3.9.0",
     "node-sass": "^4.9.3"
-  }
+  },
+  "browserslist": []
```

Comment:
What about removing this empty array and using [`gulp-json-editor`](https://www.npmjs.com/package/gulp-json-editor) instead:

    var jsonEditor = require("gulp-json-editor");

    gulp.src('themebuilder/package.json')
          //...
          .pipe(jsonEditor(browsersList))
           //...
          .pipe(gulp.dest("./dest"));